File: Codeforces/lovestory.py
def loveStory(s):
    str = "codeforces"
    count = 0
    i =0
    for value in str:
        if value != s[i]:
            count = count+1
        i = i+1  

    print(count);  


n = int(input())
for i in range(0,n):
    inp = input()
    loveStory(inp)

File: Codeforces/lovestory.cpp
#include <iostream>
using namespace std;


int loveStory(){

string s("codeforces"); //can be written as string s = "codeforces"
    string str;
    short count =0;
    cin >> str;
    for (short i = 0; i < str.length(); i++)
    {
        if (str[i] != s[i]){
            count++;
        }
    }

    
    return count;
}

int main(int argc, char const *argv[])
{
    short n;
    scanf("%hd", &n);
    for (short i = 0; i < n; i++)
    {
        cout << loveStory() << endl;
    }
    
}